TAO/TAO_IDL/driver/drv_preproc.cpp

```diff
@@ -188,6 +177,17 @@ DRV_cpp_putarg (const char *str)
             }
         }
 #endif
+      if (ACE_OS::strchr (str, ' ') && !first_quote)
```

Comment:
```suggestion
      if (ACE_OS::strchr (str, ' '))
```

I'm reasonably sure we can allow cases where there are quotes and spaces by removing this.